features/step_definitions/shared/xcode.rb

```diff
@@ -79,6 +79,21 @@
   assert_equal phase_name, build_phase.name
 end
 
+Then(/^in project (.+) the target (.+) should have the build phase (.+) with the input file (.+)$/) do |project_name, target_name, phase_name, input_file|
```

Comment:
This is something that can be unit-tested instead so I'd not have an acceptance test for this because acceptance test have a high maintenance cost compared to unit and integration tests.

Reply:
this may have been a result of my earlier feedback about extending the fixture 😅 - indeed those acceptance tests are costly, fixtures can be extended to showcase various use cases, some will have accompanying acceptance tests for explicit verification while most of the others won't. 

I appreciate the time and effort you put into this @FranzBusch